include/mtp_cmd_handler_util.h

```diff
@@ -29,6 +29,7 @@ typedef struct {
 	mtp_obj_t *obj;
 } obj_data_t;
 
+mtp_err_t _hutil_get_prop_desc(mtp_uint32 format, mtp_uint32 prop_code, void *data);
```

Comment:
All these changes here are ported from the tizen branch. Why not state it explicitly?

Reply:
> All these changes here are ported from the tizen branch. Why not state it explicitly?

Yes, changes are ported from Tizen branch. These changes adds the MTP events required by the MTP Server in Android (probably Tizen as well) for the rename & copy [currently in draft].  However, let me go through remaining comments and update the changes. Thank you